tsconfig.esnext.json

```diff
@@ -0,0 +1,24 @@
+{
+  "compilerOptions": {
+    "target":"esnext",
+    "lib": [
+      "es5",
+      "es2015",
+      "es2017",
+      "dom"
+    ],
+    "module": "commonjs",
```

Comment:
This is a little confusing.  Do you intend to have ESM output here?  If so, `module` should be `ES2020` or higher I believe.